app/views/moderations/index.html.erb

```diff
@@ -85,6 +85,7 @@
   <br /><br /><br /><br />
   <center>
     <h1>Mod Dashboard (beta)</h1>
+    <a href="/community-moderation" style="display: inline-block; font-size: 1.3em; padding: 3px 8px; border-radius: 3px;">Community moderation guide</a>
```

Comment:
I think we can move these styles into a new `.scss` file, maybe `.moderation.scss` or something.